examples/java-matter-controller/java/src/com/matter/controller/Off.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller;
+
+import com.matter.controller.commands.common.CHIPCommand;
+import com.matter.controller.commands.common.CredentialIssuerCommands;
+import com.matter.controller.commands.common.MutableInteger;
+
+public class Off extends CHIPCommand {
+  private MutableInteger mNodeId;
```

Comment:
Because this store the placeholder of the mNodeId upon registration, and its value gets populate when the the CHIPCommand is initialized, and Java does not support pass by reference for a primitive type and Integer class is not mutable.